### DISCUSSION TITLE
在版本设置中，增加阅读 “crash-reports”和“logs” 的按钮

### DISCUSSION THREAD

**Discussion by FengYanMoShen** [Meloong-Git/PCL#1160] [提问] [answered]
在版本设置中，增加阅读 “crash-reports”和“logs” 的按钮
（其实在崩溃时自动会有“导出崩溃日志”的按钮😂）

阅读 “logs” 时
时间标灰色，
[XXX/INFO]标灰色
[XXX/WARN]标黄色
[XXX/ERROR]标红色

[与HMCL很像]

**Comment by WForst-Breeze** [comment, marked as answer]
当游戏崩溃时，会有查看日志和导出错误报告的选项
![image](https://user-images.githubusercontent.com/110760354/216799554-bdb22834-5fa4-4730-882c-f02123e22d86.png)
但是如果您需要您提及的功能和优化的话，可以交个 [新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)
